src/utils/wait.ts

```diff
@@ -0,0 +1,6 @@
+// Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+// SPDX-License-Identifier: Apache-2.0
+
+export default function wait(waitTimeMs: number): Promise<void> {
```

Comment:
1) Does this need to be in a folder or should it be under src as utils.ts? 
2) Can we remove this same code here https://github.com/aws/amazon-chime-sdk-js/blob/master/test/utils.ts and use this instead :)

Reply:
For the first, the SDK structure is like level 1 are all folders and then default exports, that's why I added it as a folder. So should we keep it the same? Also, I had some discussion with Kyu in initial reviews where it seems the JS SDK currently exports types as no default exports and we do not follow that convention for functions. So, what do you suggest here?

Reply:
For the second point, its a good call, but can I do it in a separate PR as there may be lot of places to update it 😅 .

Reply:
I have added a new commit addressing these two points request you to check.